backend/libraries/chat_events/src/chat_event_internal.rs

```diff
@@ -597,6 +597,8 @@ pub struct ReplyContextInternal {
 pub struct ReplyContextInternalPrevious {
     #[serde(rename = "l")]
     pub event_list_if_other: Option<(ChatId, Option<MessageIndex>)>,
+    #[serde(rename = "c")]
+    pub chat_if_other: Option<(MultiUserChatInternal, Option<MessageIndex>)>,
```

Comment:
I added this so that upgrading multiple times doesn't lose any data.
That also makes it safer in case we forget to remove this after the next upgrades.